content/introduction/core-principles.md

```diff
@@ -0,0 +1,24 @@
+---
+title: "Core Principles"
+description: "Learn more about the core principles that make up the SweetOps methodology."
+---
+
+SweetOps is built on a foundational set of core principles that ensure the methodology is comprehensive and reusable across organizations. To fully understand what SweetOps is and how to utilize it effectively, it's critical to understand these backing principles.
+
+## 100% Open Source
+
+All of the primary technology that enables SweetOps is Open Sourced by [Cloud Posse](https://cloudposse.com) and the community under Apache 2.0. Using open source licensing is a practical, deliberate strategy. It enables SweetOps to be easily adopted, less risky, and distribution of all tools and IaC is straight forward. It protects all parties to reuse the software and fosters collaboration.
+
+## Automate Everything
+
+A important ideal from the beginning has been to ensure that all pieces of the process are automated. This includes everything from AWS account creation to configuring incident response management tooling. Ensuring that all aspects of SweetOps are automated makes processes repeatable, transparent, and reliable without having to depend on quickly outdated documentation (aka WikiOps).
+
+## Code should be Small and Composable
+
+The SweetOps infrastructure as code library, purpose built tools, and reusable catalogs are all crafted to be small, composable, and do only one thing well. This allows SweetOps to **share nothing** and supports mixing and matching reusable code to help build your unique architecture.
```

Comment:
```suggestion
The SweetOps infrastructure as code library, purpose-built tools, and reusable catalogs are crafted to be small, composable, and do only one thing well. This allows SweetOps to adapt as the technologies evolve, as well as mix and match reusable components to help build your unique architecture.
```

Reply:
See the [UNIX Philosophy](https://en.wikipedia.org/wiki/Unix_philosophy)

1. Build modular programs
2. Write readable programs
3. Use composition
4. Separate mechanisms from policy
5. Write simple programs
6. Write small programs

The first six principles are pretty spot on.

Reply:
Maybe we change this to something like: Embrace the UNIX Philosophy for the Cloud Era

We don't want one tool that does it all. We are the polar opposite to platforms like OpenShift and Rancher which attempt to do everything. We want small tools that do one thing well and then stitch them together to achieve our goals. This allows us to swap in & out tools as better ones come along.

Reply:
Will use some of this language + the header locally instead of via batch. One note, I'm trying to avoid words like "component" since that is one of the main structures within SweetOps. The more we can avoid readers misconstruing those words the better.

Reply:
@osterman updated to mash the two together. Let me know your thoughts.